src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -349,28 +347,48 @@ public boolean loadResidentList() {
 		
 		TownyMessaging.sendDebugMsg("Loading Resident List");
 		String line = null;
```

Comment:
Setting this to null is redundant, `String line;` has the exact same effect.

Reply:
Eclipse complains when it isn't initialized.